Doc/library/socketserver.rst

```diff
@@ -294,14 +288,35 @@ Server Objects
 
    .. XXX should class variables be covered before instance variables, or vice versa?
 
+   .. attribute:: address_family
+
+      The family of protocols to which the server's socket belongs.
+      Common examples are :const:`socket.AF_INET` and :const:`socket.AF_UNIX`.
+
+
    .. attribute:: allow_reuse_address
 
       Whether the server will allow the reuse of an address.  This defaults to
       :const:`False`, and can be set in subclasses to change the policy.
 
 
+   .. attribute:: allow_reuse_port
+
+      Whether the server will allow the reuse of a port. This defaults to
+      :const:`False`, and can be set in subclasses to change the policy.
+
+   .. attribute:: max_packet_size
+
+      Only for datagram sockets, e.g. UDP.
```

Comment:
Should this be `i.e.` instead of `e.g.`? I'm by no means a networking expert, but the *only* datagram socket (that I know Python supports, at least) is UDP.

Reply:
yeah that’s probably right, I always mix the two up

Reply:
It's pretty much just:

- `i.e.` -> "that is"
- `e.g.` -> "for example"

if that helps you remember :)

Reply:
AFAIK, there's only one datagram socket, namely UDP (which stands for User Datagram Protocol). Variants are only in their structure (e.g., in IPv4, UDP sockets do not have a checksum while in IPv6 they need one).